boards/opentitan/Cargo.toml

```diff
@@ -5,6 +5,10 @@ authors = ["Tock Project Developers <tock-dev@googlegroups.com>"]
 build = "build.rs"
 edition = "2018"
 
+[features]
+default = ["lowrisc/device_fpga"]
```

Comment:
What is this the default of?

Reply:
Actually I think this `default` is no longer needed, I will remove it.

Reply:
Done.